kw/platform/utils.py

```diff
@@ -34,18 +34,22 @@
 class UserAgentValidator:
     def __init__(self, value):
         self.value = value
-        self.is_valid = bool(USER_AGENT_RE.match(self.value))
+        self.is_valid = bool(self.value and USER_AGENT_RE.match(self.value))
```

Comment:
You don't need the `bool` cast I think

Reply:
But I would like it more if the value of `is_valid` was `True` or `False` and not regex match or None...

Reply:
```
In [3]: None and re.match(r".*", "asdf")                                                                                                                               

In [4]: True and re.match(r".*", "asdf")                                                                                                                               
Out[4]: <re.Match object; span=(0, 4), match='asdf'>
```

Reply:
Oh, sorry, yep, you are rgith